content-repo/extra-docs/packs/palo-alto-networks-cortex-xdr---investigation-and-response.md

```diff
@@ -62,6 +64,40 @@ After the remediation, if there are no new alerts, the playbook stops the alert
+
+## AWS IAM User Access Investigation
+The [AWS IAM user access investigation](https://xsoar.pan.dev/docs/reference/playbooks/cortex-xdr---aws-iam-user-access-investigation) playbook investigates and responds to Cortex XDR Cloud alerts where an AWS IAM user's access key is used suspiciously to access the cloud environment. 
+
+The playbook fetches data from the incident and then retrieves additional cloud alert data that was not available in the incident. It then checks if the alerts are one of the following XCLOUD supported alerts: 
+- Penetration testing tool attempt
+- Penetration testing tool activity
+- Suspicious API call from a Tor exit node.
+
+If the alert is not one of the supported alerts, the playbook ends.
+Otherwise, the incident type is set to XCLOUD and the playbook starts to collect additional information pertaining to the alert.
+
+First the source IP addresses are enriched. These are the IP addresses that are used to connect to the environment. 
+
+Then the playbook enriches information about the user who connected to the environment through the AWS IAM integration using the [AWS IAM - User enrichment](https://xsoar.pan.dev/docs/reference/playbooks/aws-iam---user-enrichment) sub-playbook. The sub-playbook lists the user access keys and retrieves information about the IAM user, including the user's creation date, path, unique ID, and ARN.  From this, it can be seen if these user keys are active and the analyst can block these keys later in the investigation if they are causing malicious activities.
+
+
+Then the playbook validates that the access key type is AKIA (which marks this as a user key). If the access key is AKIA, queries are run to retrieve the last 100 API calls made with the access key and retrieve actions performed by the user in the last 7 days. This information shows who made the call, and provides information about the IP address and data about which user was used in the request, what operation was performed, the status of the operation and on what resource it was executed.
+
+Not the investigation starts.
+First the playbook checks if there were new IP addresses that were found on the XQL queries that did not appear in the original alert and enriches them.
+Then the analyst manually reviews the results of the XQL queries from the previous steps to determine if this is a true positive event. The analyst investigates the operation performed by the access key and the user. The analyst examines the executed operations, by who it was executed, on which resource, and the operation status.
```

Comment:
Then the analyst manually reviews the results of the XQL queries from the previous steps to determine if this is a true positive event. The analyst investigates the **operation** performed by the access key and the user.
Need to be operations?

Reply:
Done.